decentraland/unity-explorer
# Feat: disk cache

## What does this PR change?

General cache to improve startup performance

## How to test the changes?

1. Launch the explorer
2. Play happy path, on second time textures and resources should load faster

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
